dagshub/common/download.py

```diff
@@ -73,6 +73,32 @@ def get_fn(bucket, path) -> bytes:
     add_bucket_downloader("s3", get_fn)
 
 
+def enable_azure_container_downloader(account_url, client=None):
+    """
+    Enables downloading storage items using the azure client, instead of going through DagsHub's server.
+    For custom clients use `enable_custom_bucket_downloader` function.
+
+    Args:
+        account_url: an azure storage account url, of the form "https://<storage-account-name>.blob.core.windows.net"
+        client: a boto3 S3 client.
+            If client isn't specified, the default parameterless constructor is used.
+            If specified, account_url is disregarded, and the client is used.
+    """
+    import io
+    if client is None:
+        from azure.storage.blob import BlobServiceClient
+        from azure.identity import DefaultAzureCredential
+        client = BlobServiceClient(account_url, credential=DefaultAzureCredential())
+
```

Comment:
Problems with this snippet: If client WAS specified, you still need to enter your account_url for some reason.

Way to solve it: make `account_url` optional, have an assert at the start that either of account_url or client is not None

The DefaultAzureCredentials looks weird to me, but I don't see a reason for exposing yet another arg, so it's fine with me